scripts/port2alias

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+import sys
+import subprocess
+from portconfig import get_port_config
+
+
+def get_platform_hwsku():
+    hwsku = None
+    platform = None
+    command = "show platform summary"
+    p = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE)
+    output = p.stdout.readlines()
+    for line in output:
+        tokens = line.split()
+        if not tokens:
+            continue
+        if tokens[0].lower() == 'hwsku:':
+            hwsku = tokens[1]
+        elif tokens[0].lower() == 'platform:':
+            platform = tokens[1]
+    return (platform, hwsku)
+
+def main():
+    (platform, hwsku) = get_platform_hwsku()
+    (port, _) = get_port_config(hwsku, platform)
+    for line in sys.stdin:
+        start = 0
+        end = 0
+        while end < len(line):
+            if line[end].isalnum():
```

Comment:
here, you have an assumption that the port name does not contain char other than alnum. For example, the port name cannot be et5_1

Reply:
What's the legal charset for sonic interface names? I thought only alnum characters are allowed.